web-components/src/components/buttons/OutlinedButton.tsx

```diff
@@ -1,22 +1,41 @@
 import { styled } from '@mui/material';
-import Button from '@mui/material/Button';
+import Button, { ButtonProps } from '@mui/material/Button';
 
-const OutlinedButton = styled(Button)(({ theme }) => ({
-  color: theme.palette.secondary.main,
-  backgroundColor: theme.palette.primary.main,
-  borderImageSlice: 1,
-  borderImageSource: 'linear-gradient(201.8deg, #4FB573, #B9E1C7)',
-  '&:hover': {
-    backgroundColor: theme.palette.secondary.light,
-    borderColor: theme.palette.secondary.light,
-    borderImageSource: 'none',
-  },
-  '&:disabled': {
-    color: theme.palette.grey[100],
-    background: theme.palette.grey[50],
-    borderImageSource: 'none',
-    borderColor: theme.palette.grey[100],
-  },
-}));
+import { ColorScheme } from '../../theme/theme.types';
 
+interface OutlinedButtonProps extends ButtonProps {
+  colorScheme?: ColorScheme;
+}
+
+const OutlinedButton = styled(Button)<OutlinedButtonProps>(
+  ({ theme, colorScheme = 'regen' }) => ({
+    color:
+      colorScheme === 'terrasos'
+        ? theme.palette.primary.contrastText
+        : theme.palette.secondary.main,
```

Comment:
hard for me to find from the design, but I think something like `rgba(var(--sc-button-text-icon-prefinance) / 1)` or as a class name `text-sc-button-text-icon-prefinance`